src/database/schema/ChannelListSchema.ts

```diff
@@ -40,6 +40,8 @@ class ChannelList implements BaseDbSchema {
 
   members: ChannelListMemberSchema[] | null | undefined;
 
+  memberUsers: UserSchema[] | null | undefined;
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The addition of `memberUsers` property in the `ChannelListSchema` is a good change. It seems to be replacing the `members` property and will now hold an array of `UserSchema` objects instead of `ChannelListMemberSchema`. This could potentially simplify data handling as we can directly access user information from the channel list.
```diff
+  memberUsers: UserSchema[] | null | undefined;
```



<!-- This is an auto-generated comment by OSS CodeRabbit -->